code/modules/food/food/snacks.dm

```diff
@@ -6975,83 +7012,584 @@
+	bitesize = 5
+	reagents.add_reagent("protein", 20)
+	reagents.add_reagent("tricordrazine", 5)
+	reagents.add_reagent("iron", 5)
+
+/obj/item/weapon/reagent_containers/food/snacks/cuttlefish
+	name = "raw cuttlefish"
+	desc = "It's an adorable squid! You couldn't possibly be thinking about eating this, right?"
+	icon = 'icons/obj/food_syn.dmi'
+	icon_state = "cuttlefish_raw"
+	nutriment_amt = 5
+
+/obj/item/weapon/reagent_containers/food/snacks/cuttlefish/Initialize()
+	. = ..()
+	bitesize = 10
+
+/obj/item/weapon/reagent_containers/food/snacks/cuttlefishcooked
+	name = "cooked cuttlefish"
+	desc = "It's a roasted cuttlefish. Rubbery, squishy, an acquired taste."
+	icon = 'icons/obj/food_syn.dmi'
+	icon_state = "cuttlefish_cooked"
+	nutriment_amt = 20
+	nutriment_desc = list("cuttlefish" = 5, "rubber" = 5, "grease" = 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/cuttlefishcooked/Initialize()
+	. = ..()
+	bitesize = 5
+	reagents.add_reagent("protein", 10)
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/monkfish
+	name = "extra large monkfish"
+	desc = "It's a huge monkfish. Better clean it first, you can't possibly eat it like this."
+	icon = 'icons/obj/food48x48.dmi'
+	icon_state = "monkfish_raw"
+	nutriment_amt = 30
+	w_class = ITEMSIZE_HUGE //Is that a monkfish in your pocket, or are you just happy to see me?
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/monkfishfillet
+	slices_num = 6
+	trash = /obj/item/weapon/reagent_containers/food/snacks/sliceable/monkfishremains
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/monkfish/Initialize()
+	. = ..()
+	bitesize = 2
+
+/obj/item/weapon/reagent_containers/food/snacks/monkfishfillet
+	name = "monkfish fillet"
+	desc = "It's a fillet sliced from a monkfish."
+	icon = 'icons/obj/food_syn.dmi'
+	icon_state = "monkfish_fillet"
+	nutriment_amt = 5
+
+/obj/item/weapon/reagent_containers/food/snacks/monkfishfillet/Initialize()
+	. = ..()
+	bitesize = 3
+	reagents.add_reagent("protein", 1)
+
+/obj/item/weapon/reagent_containers/food/snacks/monkfishcooked
+	name = "seasoned monkfish"
+	desc = "A delicious slice of monkfish prepared with sweet chili and spring onion."
+	icon = 'icons/obj/food_syn.dmi'
+	icon_state = "monkfish_cooked"
+	nutriment_amt = 10
+	nutriment_desc = list("fish" = 3, "oil" = 1, "sweet chili" = 3, "spring onion" = 2)
+
+/obj/item/weapon/reagent_containers/food/snacks/monkfishcooked/Initialize()
+	. = ..()
+	bitesize = 4
+	reagents.add_reagent("protein", 5)
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/monkfishremains
+	name = "monkfish remains"
+	icon = 'icons/obj/food_syn.dmi'
+	icon_state = "monkfish_remains"
+	desc = "The work of a madman."
+	w_class = ITEMSIZE_LARGE
+	nutriment_amt = 10
+	slice_path = /obj/item/clothing/head/fish
+	slices_num = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/monkfishremains/Initialize()
+	. = ..()
+	bitesize = 0.01 //impossible to eat
+	reagents.add_reagent("carbon", 5)
```

Comment:
Why carbon?

Reply:
It's a skellington at this point, I'm not sure how else you'd describe the taste of big bones sans (heh) marrow